skimage/util/tests/test_random_noise.py

```diff
@@ -24,7 +24,7 @@ def test_salt():
 
     # Ensure approximately correct amount of noise was added
     proportion = float(saltmask.sum()) / (cam.shape[0] * cam.shape[1])
-    assert 0.11 < proportion <= 0.15
+    assert 0.11 < proportion <= 0.16
```

Comment:
I wonder why such a low lower bound was used here and elsewhere? I realize it was this way prior to this PR, but it just seems odd that the range is so asymmetric around the requested amount.

An alternative could be to use a variable for `amount` and then set the tolerance as some fraction of that. 
```Python
assert abs(amount - proportion) < 0.05 * amount
```

Reply:
I tried it in a couple of test cases and it even passes for `0.01 * amount`

Reply:
Or maybe just a simple fixed 0.01 rather than as a fraction of amount.

Reply:
I'm not sure why the intervals were setup that way to begin with, but I symmetrized the tolerance used for the salt and pepper noise tests.  I believe the tests failed on some machines when I set the tolerance too low, but I've set it to `1e-2` for now, except for the test I believe failed on some machines at that tolerance (`5e-2`).